lib/lambda_ethereum_consensus/beacon/beacon_node.ex

```diff
@@ -58,8 +58,11 @@ defmodule LambdaEthereumConsensus.Beacon.BeaconNode do
   defp init_children(anchor_state, anchor_block) do
     Cache.initialize_cache()
 
+    time = :os.system_time(:second) |> floor()
```

Comment:
`:os.system_time` returns an integer, so no need to use floor
```suggestion
    time = :os.system_time(:second)
```